src/Config/SettingsTab.ts

```diff
@@ -331,6 +331,28 @@ export class SettingsTab extends PluginSettingTab {
                 });
             });
 
+        new Setting(containerEl)
+            .setName(i18n.t('settings.dates.nextDayStartHour.name'))
+            .setDesc(
+                SettingsTab.createFragmentWithHTML(
+                    i18n.t('settings.dates.nextDayStartHour.description') +
+                        '</br>' +
+                        this.seeTheDocumentation(
+                            'https://publish.obsidian.md/tasks/Getting+Started/Dates#Next+day+start+at',
+                        ),
+                ),
+            )
+            .addSlider((slider) => {
+                const settings = getSettings();
+                slider.setLimits(0, 23, 1);
+                slider.setValue(settings.nextDayStartHour);
+                slider.setDynamicTooltip();
+                slider.onChange(async (value) => {
+                    updateSettings({ nextDayStartHour: value });
+                    await this.plugin.saveSettings();
```

Comment:
What impact will this feature have on the "Reload the vault at midnight" facility in Tasks?

The idea of that feature is that all queries are reloaded a few seconds after midnight, so that things like "due today" are re-interpreted with the new date.

Except that with this feature set to +2, the reload at midnight will have no effect... 

And instead presumably the queries should all be reloaded at a few seconds after 2am instead - but for now, users will need to know to do that themselves manually.